doc-content/enterprise-only/installation/clustering-con.adoc

```diff
@@ -5,5 +5,5 @@ By clustering two or more computers, you have the benefits of high availability,
 
 There are several ways that you can cluster {PRODUCT} components. This document describes how to cluster the following scenarios:
 
-* <<clustering-dev-con_clustering-dev>>
-* <<clustering-ps-con_clustering-runtime-standalone>>
+* {PRODUCT} clusters in a development (authoring) environment
```

Comment:
[Links] Maybe adding the links to these sections? wdyt?

Reply:
Yes.